webapp/src/pages/Mobility.jsx

```diff
@@ -1,17 +1,27 @@
 import mobilityHeaderImage from "../assets/images/onboarding-mobility.jpg";
 import LayoutContainer from "../components/LayoutContainer";
+import { MdLink } from "../components/SumaMarkdown";
 import WaitingListPage from "../components/WaitingListPage";
 import Map from "../components/mobilitymap/Map";
 import config from "../config";
-import { t } from "../localization";
+import { mdx, t } from "../localization";
 import React from "react";
 
 export default function Mobility() {
   return config.featureMobility ? (
     <>
       <LayoutContainer top gutters>
         <h5>{t("mobility:title")}</h5>
-        <p className="text-secondary">{t("mobility:intro")}</p>
+        {mdx("mobility:intro", {
+          overrides: {
+            a: { component: MdLink },
```

Comment:
We may want to put this into `mdx` itself. To consider for the future.

Reply:
I think that's a good idea. will add it to mdx when i change the mobility content again.